src/migrations/20180516164000-endpoint-specific-apps.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const configJsonPath = process.env.CONFIG_MOUNT_POINT;
```

Comment:
I think this variable doesn't exist? The correct one is BOOT_MOUNT_POINT and then appending `/config.json` I think?

Reply:
That's what I thought, but the above points to the correct file, and `BOOT_MOUNT_POINT` was undefined in my testing.

Reply:
Ah, right, `CONFIG_MOUNT_POINT` is defined in the Dockerfile... yeah, that works fine (but for the record the one I meant was BOOT_MOUNTPOINT, without the second underscore

Reply:
Ah ok, which do you think is more correct then? For example I imagine all device types won't  use config.txt (although I know rpi+fin are the only ones which implement this kind of interface so far).

Reply:
Since we're only reading config.json here, this one is fine. If we were writing to it, going through BOOT_MOUNTPOINT would be better to allow an atomic rename.

Reply:
Ah ok, makes sense. I'll get this merged then, thanks @pcarranzav